src/screens/ContactScreen/index.js

```diff
@@ -195,9 +205,14 @@ const ContactScreen = ({navigation}) => {
     <SafeAreaView style={styles.container}>
       <StatusBar translucent={false} />
       <Header
-        title={StringConstant.chatTabHeaderCreateChatButtonText}
+        title={
+          sourceScreen === ANONYMOUS
+            ? StringConstant.chatTabHeaderCreateAnonChatButtonText
+            : StringConstant.chatTabHeaderCreateChatButtonText
+        }
         containerStyle={styles.containerStyle}
         subTitle={'Next'}
+        subtitleStyle={selectedUsers.length > 0 && styles.subtitleStyle(sourceScreen)}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The title of the header is now dynamic based on whether the source screen is `ANONYMOUS` or not. This is a good change for improving the user interface and making it more intuitive.



<!-- This is an auto-generated comment by OSS CodeRabbit -->